src/components/ScenarioParameters/ScenarioParameters.js

```diff
@@ -176,100 +189,71 @@ const ScenarioParameters = ({
     );
   };
 
-  const startParametersEdition = (event) => {
-    if (accordionSummaryExpanded) {
-      event.stopPropagation();
-    }
-    changeEditMode(true);
-  };
-
-  const [showWarningBeforeLaunchPopup, setShowWarningBeforeLaunchPopup] = useState(false);
-  const closeConfirmLaunchDialog = () => {
-    setShowWarningBeforeLaunchPopup(false);
-  };
-
   const updateBeforeLaunch = useRef(null);
 
-  const confirmAndLaunch = (event, updateBeforeLaunch_) => {
+  const scenarioLaunch = (event, updateBeforeLaunch_) => {
```

Comment:
Functions names should start with a verb, could we rename it? (we already have `startScenarioLaunch` though, I don't know what a good name would be here).
_Idem_ for function `scenarioSave` below.